src/Equinor.ProCoSys.IPO.Command/Email/SmtpService.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System.Net.Mail;
+using System.Net;
+using System;
+using System.Threading.Tasks;
+using Equinor.ProCoSys.IPO.Command.InvitationCommands;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.PersonAggregate;
+using System.IO;
+using System.Net.Mime;
+using System.Text;
+using Attachment = System.Net.Mail.Attachment;
+using System.Linq;
+using Equinor.ProCoSys.IPO.Command.InvitationCommands.CreateInvitation;
+using Equinor.ProCoSys.Auth.Authentication;
+using Microsoft.Extensions.Options;
+using Equinor.ProCoSys.Auth.Client;
+
+namespace Equinor.ProCoSys.IPO.Command.Email
+{
+    public class SmtpService : ISmtpService
+    {
+        protected readonly IOptionsMonitor<SmtpOptions> _smtpOptions;
+
+        public SmtpService(IOptionsMonitor<SmtpOptions> smtpOptions) => _smtpOptions = smtpOptions;
+
+        public async Task SendAsync(MailMessage message)
+        {
+            // SendMailAsync is not thread-safe, so to ensure it is not busy sending an mail I instantiate an new SmtpClient
+            var client = new SmtpClient(_smtpOptions.CurrentValue.Server, _smtpOptions.CurrentValue.Port);
+            client.EnableSsl = _smtpOptions.CurrentValue.EnableSSL;
+            client.Credentials = new NetworkCredential(_smtpOptions.CurrentValue.From, _smtpOptions.CurrentValue.Password);
+            await client.SendMailAsync(message);
+        }
+
+        public async Task SendSmtpWithInviteAsync(Invitation invitation, string projectName, Person organizer, string pcsBaseUrl, CreateInvitationCommand request)
+        {
+            var baseUrl = InvitationHelper.GetBaseUrl(pcsBaseUrl, invitation.Plant);
+
+            var message = new MailMessage()
+            {
+                From = new MailAddress(organizer.Email, $"{organizer.FirstName} {organizer.LastName}"),
+                Subject = InvitationHelper.GenerateMeetingTitle(invitation, projectName, request.Type,
+                            request.Type == DisciplineType.DP ? request.McPkgScope : request.CommPkgScope),
+                Body = InvitationHelper.GenerateMeetingDescription(invitation, baseUrl, organizer, projectName, true, _smtpOptions.CurrentValue.FakeEmail),
+                Priority = MailPriority.Normal
+            };
+            invitation.Participants.ToList().ForEach(y =>
+            {
+                message.To.Add(new MailAddress(y.Email, $"{y.FirstName} {y.LastName}"));
+            });
+
+            var avBody = AlternateView.CreateAlternateViewFromString(message.Body, Encoding.UTF8, MediaTypeNames.Text.Html);
+            message.AlternateViews.Add(avBody);
+
+            var inviteString = CreateInviteString(message, invitation);
+            var attachment = CreateInviteAttachment(inviteString);
+            message.Attachments.Add(attachment);
+
+            var contype = new ContentType("text/calendar");
```

Comment:
rename to contentType + use object initializer: `var contentType = new ContentType("text/calendar") { CharSet = "UTF-8" };`